thirdeye/thirdeye-frontend/app/pods/components/rootcause-select-metric-dimension/component.js

```diff
@@ -48,23 +52,54 @@ export default Component.extend({
     const filterMap = toFilterMap(toFilters(selectedUrn));
 
     if (!_.isEqual(baseUrn, baseUrnCache)) {
-      this.setProperties({ baseUrnCache: baseUrn });
+      setProperties(this, { baseUrnCache: baseUrn });
       this._fetchFilters(baseUrn, filterMap);
     }
 
-    this.setProperties({ baseUrn, filterMap });
+    setProperties(this, { baseUrn, filterMap });
   },
 
-  filters: computed('filterMap', {
-    get() {
-      const { filterMap } = this.getProperties('filterMap');
-      return JSON.stringify(filterMap);
+  inclusions: computed('filterMap', {
```

Comment:
Can all these new CP have a comment doc added to them? Explaining what they do would help understand a lot of all these `filter(t => t[1] === '=');` . Not sure what you expecting in for you to filter etc.